packages/models/src/box/box.js

```diff
@@ -946,13 +946,19 @@ boxSchema.statics.findBoxesMinimal = function findBoxesMinimal (opts = {}) {
 
 boxSchema.statics.findBoxesLastMeasurements = function findBoxesLastMeasurements (opts = {}) {
   const schema = this;
-  const { fromDate, toDate } = opts;
+  const { fromDate, toDate, full } = opts;
   const query = buildFindBoxesQuery(opts);
-
+  
   if (!fromDate && !toDate) {
-    return Promise.resolve(schema.find(query, BOX_PROPS_FOR_POPULATION)
-      .cursor({ lean: true })
-    );
+    if(full) {
```

Comment:
```suggestion
    if (full === 'true') {
```
At least add the space here please.

Have you checked if this isn't evaulating as `true`? I cannot recall if the params parser automatically converts `true` and `false` to booleans. If not this will never be false because both are not empty strings.

Reply:
Have you checked? I only looked at https://github.com/sensebox/openSenseMap-API/blob/development/packages/api/lib/controllers/statisticsController.js#L40 

I cannot remember if this works like this or not. ;)

If you would add a test testing the response this would make me happy and the pull request round ;)